src/secureCreatives.js

```diff
@@ -74,21 +98,43 @@ function sendAdToCreative(adObject, remoteDomain, source) {
 function resizeRemoteCreative({ adUnitCode, width, height }) {
   // resize both container div + iframe
   ['div', 'iframe'].forEach(elmType => {
-    let elementStyle = getElementByAdUnit(elmType).style;
-    elementStyle.width = `${width}px`;
-    elementStyle.height = `${height}px`;
+    const element = getElementByAdUnit(elmType);
+    if (typeof element !== 'undefined') {
+      if (element === null) {
+        events.emit(ERROR_SECURE_CREATIVE, {
+          msg: 'No element of type "elmType" for adUnit',
+          adUnitCode,
+          elmType,
+          width,
+          height
+        });
+      } else {
+        let elementStyle = element.style;
+        elementStyle.width = `${width}px`;
+        elementStyle.height = `${height}px`;
+      }
+    }
   });
   function getElementByAdUnit(elmType) {
-    return document
-      .getElementById(
-        find(
-          window.googletag
-            .pubads()
-            .getSlots()
-            .filter(isSlotMatchingAdUnitCode(adUnitCode)),
-          slot => slot
-        ).getSlotElementId()
-      )
-      .querySelector(elmType);
+    const element = document.getElementById(
```

Comment:
Could be because I'm not accustomed with the code behaviour, but having `element` in this function and in the caller got me confused. I'd probably rename the function with `getInnerElementByAdUnit`, and the rest would naturally follow.

Reply:
👍 I won't change the function name, in order to stay closer to the original, but I will change the names of the variables I added. Unless I find a bug here, everything will be reverted and back to un-understandable code!